.github/workflows/docs.yaml

```diff
@@ -31,6 +31,12 @@ jobs:
         files: site/content/*
         config_file: ".github/markdown_lint_config.json"
 
+    - name: Install linkinator
+      run: npm install -g linkinator
+
+    - name: Check links
+      run: make docs docs-check-links
```

Comment:
```suggestion
      run: make docs-check-links
```

Reply:
why remove `docs`? without it there's nothing under `site/public`, or what I missed?